Form-Controls/styles.css

```diff
@@ -0,0 +1,31 @@
+body {font-family: Arial, Helvetica, sans-serif;}
+* {box-sizing: border-box;}
+
+input[type=text],input[type=email] select, fieldset{
+  width: 50%;
+  padding: 12px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+  margin-top: 6px;
+  margin-bottom: 16px;
+  resize: vertical;
+}
+
+button {
+  background-color: #04AA6D;
+  color: white;
+  padding: 12px 20px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: #45a049;
+}
+
+main {
+  border-radius: 5px;
+  background-color: #f2f2f2;
+  padding: 20px;}
```

Comment:
Do you have Prettier installed?

https://syllabus.codeyourfuture.io/guides/code-style-guide#using-prettier-to-format-code-automatically